lib/server_health_check_rails/engine.rb

```diff
@@ -1,5 +1,16 @@
 module ServerHealthCheckRails
   class Engine < ::Rails::Engine
     isolate_namespace ServerHealthCheckRails
+
+    initializer 'server_health_check_rails.bypass_ssl_for_health_index_path' do |app|
```

Comment:
As I stated in another comment, my goal in this PR is to handle all health-check related configuration behind the scenes.  Our devops setup depends on this route being accessible via HTTP.  Any application of ours that uses this gem will need to configure it this way.